hack/build-test-images.sh

```diff
@@ -27,3 +27,5 @@ buildimage sti_test/sti-fake-user test/integration/images/sti-fake-user
 buildimage sti_test/sti-fake-scripts test/integration/images/sti-fake-scripts
 buildimage sti_test/sti-fake-scripts-no-save-artifacts test/integration/images/sti-fake-scripts-no-save-artifacts
 buildimage sti_test/sti-fake-no-tar test/integration/images/sti-fake-no-tar
+buildimage sti_test/sti-fake-onbuild-source test/integration/images/sti-fake-onbuild-source
+buildimage sti_test/sti-fake-onbuild test/integration/images/sti-fake-onbuild
```

Comment:
i don't see the purpose of this image?  and i'd just call the other one "sti-fake-onbuild"  instead of "sti-fake-onbuild-source" (it is the builder image which uses ONBUILD commands)